va/va.h

```diff
@@ -3437,6 +3437,25 @@ VAStatus vaMapBuffer (
     void **pbuf 	/* out */
 );
 
+/**
+ * Map data store of the buffer into the client's address space
+ * this interface could be used to convey the operation hint
+ * backend driver could use these hint to optimize the implementations
+ */
+typedef enum
+{
+    VAMapBufferFlagDefault = 0,
```

Comment:
I think we need to add a comment what default means (i.e. read|write). Also, I think we need to add a comment that these enum flags are being supported for bitwise operations.

Reply:
please, resolve

Reply:
I suggest this text:
```
/** \brief VAMapBufferFlagDefault is used when there are no flag specified
 * Same as VAMapBufferFlagRead | VAMapBufferFlagWrite.
 */

```

Reply:
they are enum values , not macro, it is why I dont use  VAMapBufferFlagRead | VAMapBufferFlagWrite, or next extended flag must set value to 0x4?

Reply:
> next extended flag must set value to 0x4?

Maybe we will just use `#define VAMapBufferFlagDefault` 0 and so on then?